src/SFA.DAS.EmployerCommitmentsV2.Web/Validators/SendNewTrainingProviderRequestValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Validators
+{
+    public class SendNewTrainingProviderRequestValidator : AbstractValidator<SendNewTrainingProviderRequest>
+    {
+        public SendNewTrainingProviderRequestValidator()
+        {
+            RuleFor(r => r.AccountHashedId).NotEmpty();
```

Comment:
do we not need to validate the non-hashed AccountId and ApprenticeshipId? AccountId looks as if it is never used, can it be removed from the request class?